examples/compose/plot_transformed_target.py

```diff
@@ -43,21 +38,24 @@
     density_param = {'normed': True}
 
 ###############################################################################
-# A synthetic random regression problem is generated. The targets ``y`` are
-# modified by: (i) translating all targets such that all entries are
-# non-negative and (ii) applying an exponential function to obtain non-linear
-# targets which cannot be fitted using a simple linear model.
+# A synthetic random regression dataset is generated. The targets ``y`` are
+# modified by:
+#
+#   1. translating all targets such that all entries are
+#      non-negative (by adding the absolute value of the lowest ``y``) and
+#   2. applying an exponential function to obtain non-linear
+#      targets which cannot be fitted using a simple linear model.
 #
 # Therefore, a logarithmic (`np.log1p`) and an exponential function
 # (`np.expm1`) will be used to transform the targets before training a linear
 # regression model and using it for prediction.
 
 X, y = make_regression(n_samples=10000, noise=100, random_state=0)
-y = np.exp((y + abs(y.min())) / 200)
+y = np.expm1((y + abs(y.min())) / 200)
```

Comment:
@glemaitre I changed this to `expm1` as you stated above in the text. I just want to make sure this is correct.

Reply:
Yes it was a mistake Thanks to catch this up